src/main/java/seedu/foodrem/logic/parser/CliSyntax.java

```diff
@@ -11,4 +11,11 @@ public class CliSyntax {
     public static final Prefix PREFIX_ITEM_UNIT = new Prefix("u/");
     public static final Prefix PREFIX_ITEM_BOUGHT_DATE = new Prefix("bgt/");
     public static final Prefix PREFIX_ITEM_EXPIRY_DATE = new Prefix("exp/");
+
+    /* Definitions for Sort Command options */
```

Comment:
Just a question (but I don't think it needs to be refactored right now), since we have many prefixes that overlap, should we split them up (like what is done here). Or find a better way to rename and reuse them. 

For example should we just use NAME for all item, tag and sorting prefixes? Or split them for clarity/verbosity

Reply:
Hmm I'm not 100% sure, I split them up for now but as you mentioned, it can be easily refactored. I'll open this as an issue #182